honeybee_radiance_folder/folderutil.py

```diff
@@ -8,17 +8,17 @@ class NonApertureState(object):
     """A state for a dynamic non-aperture geometry from Radiance files.
 
     Args:
-        name (str): Optional human-readable name for the state. Can be None.
+        identifier (str): Human-readable identifier for the state.
         default (str): Path to file to be used for normal representation of the geometry.
         direct (str): Path to file to be used for direct studies.
 
     Properties:
-        * name
+        * identifier
         * default
         * direct
     """
-    def __init__(self, name, default, direct):
-        self.name = name
+    def __init__(self, identifier, default, direct):
```

Comment:
Should the name of this class be changed to `SceneState` since anything that uses this class is a part of the `scene_dynamic` folder, right?

Reply:
Or, in honeybee-radiance we call this a `ShadeState`. I realize that you might want to use a different naming convention from honeybee since this is all supposed to be specific to Radiance and not the conventions of the Honeybee Model . But I think you should just either follow the own convention you set up here (calling is `SceneState`) or follow the honeybee-core convention (calling it a `ShadeState`). Right now, this name doesn't seem to com from anywhere except an obsolete naming convention.

Reply:
Also, if you go with `SceneState`,  it might be good to say in the docstring that this object parallels the honeybee-radiance `ShadeState` just to make it clear to people familiar with honeybee-radiance that these refer to the same thing.

Reply:
I was just lazy in changing it. This is the old name based on the older folder structure. I will do SceneState to match the folder itself and add a comment about shadeState.